wiki/Performance_points/id.md

```diff
@@ -0,0 +1,134 @@
+
+*Lihat juga: [Akurasi](/wiki/Accuracy)*
+
+*Akurasi* adalah sebuah pengukuran persen dari kemampuan pemain untuk mengenai [hit objek](/wiki/Hit_object) pada waktunya; Berhubungan dengan algoritma pp, itu juga adalah sebuah nilai inti yang digunakan untuk mengevaluasi kinerja individu pemain pada sebuah beatmap.
+
+Skor dengan jumlah akurasi yang tinggi itu dipertimbangkan oleh algoritma tersebut sebagai yang paling mengagumkan, dan juga akan memberikan skor yang sangat besar oleh karena ini. Sebuah play [full combo](/wiki/Glossary#fc) yang diraih dengan 80% akurasi kadang-kadang bisa hanya senilai 2/3 dari skor yang diraih dengan 95% akurasi. Karena algoritma ini sangat bergantung pada akurasi, mod seperti Hidden, Hard Rock dan Flashlight itu dipertimbangkan untuk meningkatkan jumlah pp tersebut dengan akuraai yang tinggi secara signifikan.
+
+### Strain
+
+*Strain* adalah nilai inti yang mempertimbangkan berapa kali, dan untuk berapa lama, seorang pemain dikenakan bagian dengan intensitas tinggi dalam beatmap tertentu.
+
+Bagian dengan kecepatan sangat tinggi atau [pola](/wiki/Beatmap/Pattern) yang sangat sulit dalam suatu beatmap akan secara signifkan meningkatkan nilai strainnya. Sebagai contoh, beatmap dengan banyak [stream](/wiki/Beatmap/Pattern/Stream) atau bagian jump yang cepat akan mempunyai nilai strain yang tinggi, dan akibatnya meningkatkan jumlah pp yang diperoleh pada beatmap itu.
+
+<!-- consult reg contributors before moving FAQ to a new location-->
+
+## FAQ
+
+### Dimanakah saya bisa melihat peringkat performa?
+
+**Peringkat performa untuk semua pemain dapat ditemukan di [halaman peringkat](https://osu.ppy.sh/p/pp).**
+
+Kamu juga bisa menuju laman peringkat melalui panel dropdown `peringkat` pada desain web yang lama, dan memilih opsi `performa`.
+
+### Bagaimana cara saya menaikkan peringkat dan jumlah total pp?
+
+**Performamu sebagian besar diperingkatkan berdasarkan skor pada map-map individu.**
+
+Cara yang terbaik untuk meningkatkan skill kamu adalah mencoba untuk mendapatkan skor bagus di map-map yang sulit atau bermain berbagai macam jenis beatmap.
+
+Pertimbangkan tip-tip berikut ini:
+
+- Bermainlah secara efisien dan cari tahu cara bermain yang cocok bagimu.
+- Fokus dalam mendapatkan beberapa skor yang mengagumkan daripada "farming" ratusan skor yang biasa saja.
+- Berusahalah untuk meningkatkan akurasimu. Bahkan 1% saja sudah membuat perbedaan yang sangat besar.
+- Berusahalah untuk mencapai kombo yang lebih banyak. Full Combo (FC) atau skor [SS](/wiki/Glossary/grade) dapat memberikan jumlah skor yang sangat banyak.
+
+### Kenapa saya tidak mendapatkan seluruh jumlah pp dari map yang saya telah mainkan? 
+
+Performance points menggunakan sistem pemboboton, yang berarti skor paling tinggimu memberikan 100% jumlah pp-nya, dan setiap skor yang kamu dapatkan setelahnya memberikan lebih sedikit secara bertahap.
+
+Kamu bisa mempelajari lebih lanjut tentang sistem pemboboton [di atas](#weightage-system). 
+
```

Comment:
this is still not resolved btw

```
### Berapa banyak pp bonus yang diberikan untuk mendapatkan skor dalam banyak beatmap Ranked?
```
that on line 108, then another blank line on line 109

Reply:
somehow missed it, thanks for reminding